app/controllers/auth_controller.rb

```diff
@@ -1,6 +1,8 @@
 class AuthController < ApplicationController
   helper :auth
 
+  @@attempts = 0
```

Comment:
@yangsong8 : What we were trying to achieve here was if user enters the wrong password for three consecutive times, then he will be shown with google recaptcha. This was to prevent from brute force attack.